syntax = "proto3";

package camera;

service CameraService {
  // 定義流式上傳影像資料
  rpc StreamFrames (stream Frame) returns (Status);
}

message Frame {
  bytes image_data = 1; // 影像數據
  int32 width = 2; // 影像寬度
  int32 height = 3; // 影像高度
}

message Status {
  string message = 1; // 狀態消息
}
